catalogs/catalog-lakehouse-paimon/src/test/java/com/datastrato/gravitino/catalog/lakehouse/paimon/integration/test/CatalogPaimonBaseIT.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.lakehouse.paimon.integration.test;
+
+import com.datastrato.gravitino.Catalog;
+import com.datastrato.gravitino.NameIdentifier;
+import com.datastrato.gravitino.SchemaChange;
+import com.datastrato.gravitino.SupportsSchemas;
+import com.datastrato.gravitino.client.GravitinoMetalake;
+import com.datastrato.gravitino.integration.test.container.ContainerSuite;
+import com.datastrato.gravitino.integration.test.util.AbstractIT;
+import com.datastrato.gravitino.integration.test.util.GravitinoITUtils;
+import com.google.common.collect.Maps;
+import java.util.Collections;
+import java.util.Map;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public abstract class CatalogPaimonBaseIT extends AbstractIT {
+
+  protected static final ContainerSuite containerSuite = ContainerSuite.getInstance();
+
+  private static final String provider = "lakehouse-paimon";
+  private static final String catalog_comment = "catalog_comment";
+  private static final String schema_comment = "schema_comment";
+  private String metalakeName = GravitinoITUtils.genRandomName("paimon_it_metalake");
+  private String catalogName = GravitinoITUtils.genRandomName("paimon_it_catalog");
+  private String schemaName = GravitinoITUtils.genRandomName("paimon_it_schema");
+  private GravitinoMetalake metalake;
+  private Catalog catalog;
+
+  @BeforeAll
+  public void startup() {
+    containerSuite.startHiveContainer();
+    createMetalake();
+    createCatalog();
+    createSchema();
+  }
+
+  @AfterAll
+  public void stop() {
+    clearTableAndSchema();
+    metalake.dropCatalog(catalogName);
+    client.dropMetalake(metalakeName);
+  }
+
+  @AfterEach
+  private void resetSchema() {
+    clearTableAndSchema();
+    createSchema();
+  }
+
+  protected abstract Map<String, String> initPaimonCatalogProperties();
+
+  private void clearTableAndSchema() {}
+
+  private void createMetalake() {
+    GravitinoMetalake[] gravitinoMetalakes = client.listMetalakes();
+    Assertions.assertEquals(0, gravitinoMetalakes.length);
```

Comment:
It works because every IT start with a fresh env now, we plan to reuse the env to reduce IT time, so I think we could remove the check, Paimon IT only cares for Paimon metalake

Reply:
ok

Reply:
done